src/main/java/seedu/flashnotes/logic/commands/ExitCommand.java

```diff
@@ -1,6 +1,6 @@
-package seedu.address.logic.commands;
+package seedu.flashnotes.logic.commands;
 
-import seedu.address.model.Model;
+import seedu.flashnotes.model.Model;
 
 /**
  * Terminates the program.
```

Comment:
Perhaps the "Address Book" in MESSAGE_EXIT_ACKNOWLEDGEMENT could be changed to "Flash Notes"?